guides/getting_started.md

```diff
@@ -0,0 +1,113 @@
+## Getting Started with Solidus
+
+This article will explain how to setup a new Rails application with Solidus installed and give you
+hints at how to start customizing your store. We assume familiarity with Ruby and Rails, but will
+link to further resources to help you understand what we are doing.
+
+## Installing Solidus
+
+This chapter will guide you through installing Solidus for the first time.
+
+
+### Installing Image Magick
+
+Solidus also uses the ImageMagick library for manipulating images. Using this library allows for automatic resizing of product images and the creation of product image thumbnails. ImageMagick is not a Rubygem and it can be a bit tricky to install. There are, however, several excellent sources of information on the Web for how to install it. A basic Google search should help you if you get stuck.
+
+If you are using OSX, a recommended approach is to install ImageMagick using [Homebrew](http://brew.sh/). This can be done with the following command:
+
+```bash
+$ brew install imagemagick
+```
+
+If you are using Unix or Windows check out [Imagemagick.org](http://www.imagemagick.org/) for more detailed instructions on how to setup ImageMagick for your particular system.
+
+
+### Installing Ruby
+
+Installing Ruby varies widely from architecture to architecture. There is a good overview on how to
+install Ruby on the [Ruby homepage](https://www.ruby-lang.org/en/documentation/installation/).
+
+Please make sure your ruby version is at least 2.2.2:
+
+```
+$ ruby -v
+ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
+```
+
+### Installing Rails
+
+Ruby comes with the package manager `rubygems`. If your Ruby is setup correctly, this should be as easy as running the following command:
+
+```
+$ gem install rails bundler
+```
+
+### Generating your host app
+
+It's important to understand that Solidus is a [Rails engine](http://guides.rubyonrails.org/engines.html), and that your host app simply requires that engine. The first step for installing Solidus is therefore generating a vanilla Rails application.
+
+Solidus will not work well with the `sqlite` database, and it runs best with the `postgresql` database.
+The ecosystem in general uses the `rspec` testing framework, so we skip the standard `minitest` installation. To generate your host app with these options, run the following command:
+
+```
+rails new my-shop -d postgresql --skip-test
+```
+
+Switch into your newly generated app directory:
+
+```
+cd my-shop
+```
+
+Now, run the tasks to create your development and test databases. If this does not work, please consult the postgres installation instructions for your architecture and the Rails Guide on [Configuring Rails applications](http://edgeguides.rubyonrails.org/configuring.html#rails-general-configuration).
+
+```
+bundle exec rake db:create db:migrate
```

Comment:
Sure the `db:create` is mandatory. I meant the `db:migrate`, sorry or being unclear.